Google-Summer-of-Code-2025.md

```diff
@@ -345,6 +345,7 @@ If you need clarification on any of these ideas, feel free to open a thread in G
 
 4.2. [Platform parameters dashboard](#42-platform-parameters-dashboard)
 
+4.3. [Android lint infrastructure & fixes](#43-android-lint-infrastructure-&-fixes)
```

Comment:
@BenHenning Would it be OK to change "&" to "and"? Not strictly required, but I get thrown off a bit by the symbols when doing the gsoc admin spreadsheet prep since it's unusual for us to have `&` in project titles.